frontend/src/APIClients/queries/StoryQueries.ts

```diff
@@ -137,13 +137,23 @@ export type StoryTranslation = {
   numApprovedLines: number;
 };
 
-export const buildStoriesQuery = () => {
-  // TODO: build out when filters added
+export const buildStoriesQuery = (
+  language?: string,
+  level?: number,
+  stage?: string,
+  storyTitle?: string,
+) => {
+  let queryParams = language ? `language: "${language}", ` : "";
+  queryParams += level ? `level: ${level}, ` : "";
+  queryParams += stage ? `stage: "${stage}", ` : "";
+  queryParams += storyTitle ? `storyTitle: "${storyTitle}", ` : "";
+  queryParams = queryParams === "" ? queryParams : `( ${queryParams} )`;
+
   return {
     fieldName: "storyTranslations",
     string: gql`
       query StoryTranslations {
-        storyTranslations {
+        storyTranslations ${queryParams} {
```

Comment:
Not enclosed in parenthesis because if we have an empty set of `queryParams` and create a query like `storyTranslations () { ... } ` GQL throws an error

Reply:
lgtm 👍  lines 151 - 155 are pretty satisfying lookin 😍